nexmark-flink/src/main/resources/conf/flink-conf.yaml

```diff
@@ -32,78 +33,22 @@ env.java.opts: -verbose:gc -XX:NewRatio=3 -XX:+PrintGCDetails -XX:+PrintGCDateSt
 env.java.opts.jobmanager: -Xloggc:$FLINK_LOG_DIR/jobmanager-gc.log -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=2 -XX:GCLogFileSize=512M
 env.java.opts.taskmanager: -Xloggc:$FLINK_LOG_DIR/taskmanager-gc.log -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=2 -XX:GCLogFileSize=512M
 
-#==============================================================================
-# Job schedule & failover
-#==============================================================================
-
-# Restart strategy related configuration
-restart-strategy: fixed-delay
-restart-strategy.fixed-delay.attempts: 2147483647
-restart-strategy.fixed-delay.delay: 10 s
-
-# Max task attempts to retain in JM
-jobmanager.execution.attempts-history-size: 100
-
-#==============================================================================
-# Resources & Slots
-#==============================================================================
-
-# Timeout (ms) for an idle TM to be released.
-slotmanager.taskmanager-timeout: 600000
-
-#==============================================================================
-# Network
-#==============================================================================
-
-# Number of extra network buffers to use for each outgoing/incoming gate
-# (result partition/input gate).
-taskmanager.network.memory.floating-buffers-per-gate: 256
-
-# The number of buffers available for each external blocking channel.
-# Will change it to be the default value later.
-taskmanager.network.memory.buffers-per-external-blocking-channel: 16
-
-# The maximum number of concurrent requests in the reduce-side tasks.
-# Will change it to be the default value later.
-task.external.shuffle.max-concurrent-requests: 512
-
-# Whether to enable compress shuffle data when using external shuffle.
-# Will change it to be the default value later.
-task.external.shuffle.compression.enable: true
-
-# Maximum backoff time (ms) for partition requests of input channels.
-taskmanager.network.request-backoff.max: 300000
-
 #==============================================================================
 # State & Checkpoint
 #==============================================================================
 
 state.backend: rocksdb
-state.checkpoints.dir: file:///path/to/checkpoint
+state.checkpoints.dir: file:///path/to/checkpoint # for example, hdfs://benchmark01/checkpoint
```

Comment:
IIRC, Flink YAML doesn't work well with inline comments, could you move the comment above the line?

Reply:
Resolved by a fixup commit.